web/src/app/admin/keys/openai/page.tsx

```diff
@@ -49,14 +57,167 @@ const ExistingKeys = () => {
   );
 };
 
+const LLMOptions = () => {
```

Comment:
Is there a preferred place to put this component?

Reply:
Here is fine! Or as a named file in the `keys/openai` dir is also pretty standard across the pages